### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by NGHDev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1 +1 @@
-
```
Comment (gadnandev) [inline index.html:1]: Could you write Welcome instead of OK

Comment (NGHDev) [reply index.html:1]: Sure, I will do it

**Review by gadnandev — CHANGES_REQUESTED** [review]
Please, do it as mentioned in the review

**Review by gadnandev — APPROVED** [review]
Well done Nghdev!

**Pull request merged**